components/emoji_picker/emoji_picker.tsx

```diff
@@ -23,6 +25,8 @@ import EmojiPickerCurrentResults from 'components/emoji_picker/components/emoji_
 
 import type {PropsFromRedux} from './index';
 
+const CUSTOM_EMOJI_SEARCH_THROTTLE_TIME_MS = 1000;
```

Comment:
Do you mind moving the constants inside of `components/emoji_picker/constants/index.ts`

Reply:
Done

Reply:
Oh we didnt move this to constants folder :P can you please check

Reply:
Ohh sorry my mistake. It should be fixed now